src/main/java/net/dv8tion/jda/api/requests/restaction/ChannelAction.java

```diff
@@ -185,6 +185,23 @@
     @CheckReturnValue
     ChannelAction<T> setSlowmode(int slowmode);
 
+    /**
+     * Sets the news flag for the new TextChannel
```

Comment:
Explain this further, this is too little information to understand what "news" means. For example you can link to `TextChannel#isNews` or use it for reference for what needs to be mentioned

Reply:
is adding TextChannel#isNews to "see also" enough? i think yes, as it says what news channel mean.

Reply:
No it isn't because see also is for related methods. This requires an explanation in the description.